pkg/fqdn/service/service.go

```diff
@@ -306,6 +312,12 @@ func (s *FQDNDataServer) sendAndRecvAckForDNSPolicies(stream pb.FQDNData_StreamP
 			}
 			endpointInfos = append(endpointInfos, endpointInfo)
 		}
+		if len(prefixWithoutEndpoint) > 0 {
+			// Add IPs without endpoint info i.e. cidrs that do not map to any endpoint
+			endpointInfos = append(endpointInfos, &pb.EndpointInfo{
```

Comment:
This isn't an endpoint. Why does it have IPs?

Reply:
Addressed this by adding a separate table for identity mappings.